common/js_handle.go

```diff
@@ -79,7 +80,13 @@ func (h *BaseJSHandle) AsElement() *ElementHandle {
 
 // Dispose releases the remote object.
 func (h *BaseJSHandle) Dispose() {
+	fmt.Println(">>>>>>: BaseJSHandler Dispose")
 	if err := h.dispose(); err != nil {
+		if errors.Is(err, context.Canceled) {
+			fmt.Println(">>>>>>: Skipping context canceled error on Dispose", err)
+			return
+		}
```

Comment:
I'm [happy](https://github.com/grafana/xk6-browser/issues/1089#issuecomment-2056669729) with this approach 👍